rest-service/manager_rest/rest/resources_v1/events.py

```diff
@@ -51,38 +54,98 @@ class Events(SecuredResource):
     DEFAULT_SEARCH_SIZE = 10000
 
     @staticmethod
-    def _build_select_query(_include, filters, pagination, sort):
+    def _apply_range_filters(query, model, range_filters):
+        """Apply range filters to query.
+
+        :param query: Query in which the filtering should be applied
+        :type query: :class:`sqlalchemy.orm.query.Query`
+        :param model: Model to use to apply the filtering
+        :type model:
+            :class:`manager_rest.storage.resource_models.Event`
+            :class:`manager_rest.storage.resource_models.Log`
+        :param range_filters: Range filters passed as a request argument
+        :type range_filters: dict(str, dict(str))
+        :returns: Query with filtering applied
+        :rtype: :class:`sqlalchemy.orm.query.Query`
+
+        """
+        for field, range_filter in range_filters:
+            # Drop `@` prefix for compatibility
+            # with old Elasticsearch based implementation
+            field = field.lstrip('@')
+            if hasattr(model, field):
```

Comment:
At first I though that just ignoring that filtering criteria would be a good enough, but now I think that indeed an error is better to let the user know that there was a problem which was maybe a typo in the request.